Astronomical Data Archives (ESG WP5) (#2174)
galaxyproject/galaxy-hub: content/news/2023-09-07-esg-wp5-astronomy-archives/index.md

---
title: 'Connecting Astronomical Data Archives with Galaxy'
date: "2023-09-25"  

> tease: "Enabling astronomers to search and download data from from IVOA-complant archives directly within Galaxy"

Reviewer: ```suggestion
tease: "Enabling astronomers to search and download data from IVOA-compliant archives directly within Galaxy"
```

hide_tease: false
tags: [esg-wp5, esg]

> subsites: [all-eu, esg]

Reviewer: ```suggestion
subsites: [all-eu, esg, all]
```

main_subsite: eu
---

Astronomical observations are diverse and unique, maintaining their scientific value for centuries. This lead the astronomical community to develop traditions and practices of data preservation and interoperability in Astronomical Data Archives. Modern astronomical observatories produce large, nearing exoscale, amounts of data, 
posing new challenges which can be addressed by web-based platforms bringing analysis code to the data. Possibility to discover and analyse data from these archives is crucial for making Galaxy adopted in astronomocal community.

> As part of the effort to bring astronomical workflows to Galaxy within EuroScienceGateway project [WP5](https://galaxyproject.org/projects/esg/news/?tag=esg-wp5), teams at [EPFL](https://www.epfl.ch/en/) and [UPC](https://apc.u-paris.fr/APC_CS/) developed [a tool](https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fastroteam%2Fastronomical_archives%2Fastronomical_archives%2F0.9.0) which relies on Table Access Protocol ([TAP](https://wiki.ivoa.net/twiki/bin/view/IVOA/TableAccess)) developed by  International Virtual Observatory Alliance ([IVOA](https://www.ivoa.net/)) to allow user to:

Reviewer: ```suggestion
As part of the effort to bring astronomical workflows to Galaxy within EuroScienceGateway project [WP5](https://galaxyproject.org/projects/esg/news/?tag=esg-wp5), teams at [EPFL](https://www.epfl.ch/en/) and [UPC](https://apc.u-paris.fr/APC_CS/) developed [a tool](https://usegalaxy.eu/root?tool_id=astronomical_archives) which relies on Table Access Protocol ([TAP](https://wiki.ivoa.net/twiki/bin/view/IVOA/TableAccess)) developed by  International Virtual Observatory Alliance ([IVOA](https://www.ivoa.net/)) to allow user to:
```

We could use the short tool IDs here. This way you get also always the latest version.

* Choose a specific archive from a list of 138 different archives or query multiple archives matching a keyword
* Run custom [ADQL](https://www.ivoa.net/documents/ADQL/) TAP queries or use the IVOA table query builder
* Preview discovered resources in an easy-to-explore HTML
* Obtain a CSV file containing resource URLs matching given query    
* Download matching files directly into Galaxy for futher analysis

<div class="center">
<div class="img-sizer" style="width: 100%">

> ![Selection of APERTIF radio telescope images found at ASTRON through IVOA TAP protocol with a Galaxy tool](galaxy-ivoa-astron.png)</div>  

Reviewer: APERTIF? You guys have great names :)

Author: Well we can't really use astronomical names, it would lead to confusion! :)

<figcaption>

> Selection of <a href="https://science.astron.nl/sdc/astron-data-explorer/data-releases/apertif-dr1/">APERTIF</a> radio telescope observation data found at <a href="https://science.astron.nl/sdc/astron-data-explorer/">ASTRON</a> archive through IVOA TAP protocol with a <a href="https://usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fastroteam%2Fastronomical_archives%2Fastronomical_archives%2F0.9.0">Galaxy tool</a>

Reviewer: ```suggestion
Selection of <a href="https://science.astron.nl/sdc/astron-data-explorer/data-releases/apertif-dr1/">APERTIF</a> radio telescope observation data found at <a href="https://science.astron.nl/sdc/astron-data-explorer/">ASTRON</a> archive through IVOA TAP protocol with a <a href="https://usegalaxy.eu/root?tool_id=astronomical_archives">Galaxy tool</a>
```
